### DISCUSSION TITLE
can i generate access token for forgot password using hydra directly with custom claims ,

### DISCUSSION THREAD

**Discussion by anfaas1618** [ory/kratos#2830] [Q&A] [answered]
I am trying  to integrate forgot password functionality in my ecomm app, but to generate access token i am needing to go through process of oauth2 , i want to create access token when someone enters his email in forgot password and send that token to his email id.

**Comment by vinckr** [comment, marked as answer]
Hello @anfaas1618 
User/account management is out of scope for Hydra, you can use Ory Kratos, that has a [forgot password](https://www.ory.sh/docs/kratos/self-service/flows/account-recovery) flow or a similar solution.
